data/yam/autoyast/support_images/sles12sp5_install_default_pattern_aarch64.xml.ep

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+    <suse_register>
+        <do_registration config:type="boolean">true</do_registration>
+        <reg_code><%= $get_var->('SCC_REGCODE') %></reg_code>
```

Comment:
Here should change to <reg_code>{{SCC_REGCODE}}</reg_code>

Reply:
changed.

Reply:
OK